src/sonic-py-common/sonic_py_common/syslogger.py

```diff
@@ -4,13 +4,19 @@
 import socket
 import sys
 
+# customize python logging to support notice logger
+logging.NOTICE = logging.INFO + 1
```

Comment:
How do you ensure this integer is not used by vanilla `logging` library?

Reply:
Hi @qiluo-msft , vanilla `logger` library supports customized log level. Here is what I copied from python official code:
```python
#---------------------------------------------------------------------------
#
# Default levels and level names, these can be replaced with any positive set
# of values having corresponding names. There is a pseudo-level, NOTSET, which
# is only really there as a lower limit for user-defined levels. Handlers and
# loggers are initialized with NOTSET so that they will log all messages, even
# at user-defined levels.
#

CRITICAL = 50
FATAL = CRITICAL
ERROR = 40
WARNING = 30
WARN = WARNING
INFO = 20
DEBUG = 10
NOTSET = 0
```

Reply:
Hi @qiluo-msft , any further comment?

Reply:
To ensure future proof, in case vanilla `logging` library suddently change behavior, could you add a testcase to check? Just logging a notice message and see it is captured as a notice syslog.